src/server/system_services/account_server.js

```diff
@@ -793,6 +793,7 @@ async function update_external_connection(req) {
             endpoint: connection.endpoint,
             cp_code: connection.cp_code,
             auth_method: connection.auth_method,
+            azure_log_access_keys: connection.azure_log_access_keys,
```

Comment:
I'll move the [old PR discussion](https://github.com/noobaa/noobaa-core/pull/7308#discussion_r1258238501) to here, I def agree with not printing secrets (get_account_info()), but we do return identity so we also need to return the ids from the azure_log_access_keys object

```
azure_log_access_keys: {
            properties: {
                azure_tenant_id -> should be printed to the user
                azure_client_id -> should be printed to the user
                azure_client_secret ,
                azure_logs_analytics_workspace_id  -> should be printed to the user
            }
 }
```

what's the point of letting the user update these properties without him knowing what he put there in the first place

Reply:
@Neon-White do we want to treat all of these the same as secret_key, or does any of these fields should be returned to identify the credentials? 
e.g., for azure connections, we use the Account Name as the "identity" (`AWS_ACCESS_KEY_ID` for S3 connections). should any of these new fields be returned as part of the account information?